cypress/e2e/pages/storyPage/index.cy.js

```diff
@@ -10,14 +11,219 @@ import {
   testsThatNeverRunDuringSmokeTestingForCanonicalOnly,
 } from './testsForCanonicalOnly';
 
-const testsForPage = {
-  pageType: 'storyPage',
+const pageType = 'storyPage';
+const tests = [
```

Comment:
I'd like to refactor these tests & consolidate them into 2 - AMP + Canonical. Let's work on this together.